Assets/Scripts/Movement/MovingDirection.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using UnityEngine;
+using Assets.Scripts.Helpers;
+using System.Linq;
+using Assets.Scripts;
+
+public class MovingDirection : MonoBehaviour
```

Comment:
Change class and script name to OwlmanMovingDirection. Change all occurrences in code.